extensions/git/src/git.ts

```diff
@@ -2714,6 +2719,34 @@ export class Repository {
 		return parseGitStashes(result.stdout.trim());
 	}
 
+	async getWorktrees(): Promise<Worktree[]> {
+		const worktrees: Worktree[] = [];
+
+		worktrees.push(...await this.getWorktreesFS());
```

Comment:
`    return await this.getWorktreesFS();
`

instead of assignment and then returning